src/background/modules/ContextMenu.js

```diff
@@ -40,7 +24,8 @@ function notification(title, message) {
  */
 function fallback(text, fieldId) {
     navigator.clipboard.writeText(text);
-    notification(`📋 Press ${pasteSymbol}-V`, `Add-ons in Thunderbird are currently unable to access the “${fieldId.startsWith("compose") ? fieldId.slice("compose".length) : fieldId}” field directly, so the transformed text has been copied to your clipboard.\nPlease press ${pasteSymbol}-V to do the transformation.`);
+    // This will need to be localized
```

Comment:
Feel free to add todos as real `TODO` comments. AFAIK I also I had a not or so which would notify you in this or if you have such a one.

Reply:
Of course better fix it directly.

Reply:
OK, will do... I will leave the localization to you.